src/components/shared/Slider/index.tsx

```diff
@@ -1,6 +1,30 @@
+<<<<<<< HEAD
```

Comment:
There is conflict. Could you resolve it?

Reply:
I must have not saved my file after merging it! Thank you for the quick check